src/cljs/commiteth/manage_payouts.cljs

```diff
@@ -57,6 +57,19 @@
                        (:claims bounty))]
             [claim-card bounty claim]))))
 
+(defn bounty-stats [{:keys [paid unpaid]}]
+  (let [sum-dollars (fn [bounties]
+                      (reduce + (map #(js/parseFloat (:value_usd %)) bounties)))]
```

Comment:
I think it would be better to calculate these totals in a subscription instead, so that we are not doing computations here.

Reply:
I considered that and didn't do it because this seemed simple enough but you're probably right, will change. :)